cla-backend-go/gerrits/repository.go

```diff
@@ -80,3 +81,32 @@ func (repo repo) GetProjectGerrits(projectID string) ([]*models.Gerrit, error) {
 	}
 	return out, nil
 }
+
+func (repo *repo) UnassignProject(gerritID string) error {
```

Comment:
Harold - can we change this to a delete record? I don't think we need to keep the record.

Rename from Unassign to Delete:

func DeleteProjectGerrits(projectID string) error